typescript/simple-mercari-web/src/App.css

```
@@ -15,35 +17,84 @@
 
 .Listing {
   background-color: #222427;
-  min-height: 8vh;
+  padding: 16px;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: left;
-  font-size: calc(10px + 1vmin);
+  justify-content: center;
+  font-size: 16px;
   color: white;
 }
 
-.ItemList {
-  background-color: #222427;
-  min-height: 8vh;
+form {
+  width: 100%;
+  max-width: 800px;
   display: flex;
-  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 50px;
   align-items: center;
-  justify-content: left;
-  font-size: calc(10px + 1vmin);
-  color: white;
+  justify-content: center;
+}
+
+input {
+  padding: 10px;
+  border-radius: 5px;
+  font-size: 14px;
+  background-color: white;
+  color: black;
+}
+
+.name,
+.text,
+.file {
```

Instead of one CSS class for each field, you could instead:

```suggestion
input {
```

or

```suggestion
input[type="text"],
input[type="file"] {
```

---

I fixed.